docs/questions/query-builder/expressions/concat.md

```diff
@@ -4,36 +4,49 @@ title: Concat
 
 # Concat
 
-`concat` joins text data (strings) from two or more columns.
+`concat` concatenates data from two or more columns or values, and returns a string.
 
-| Syntax                        | Example                                  |
-|-------------------------------|------------------------------------------|
-| `concat(value1, value2, ...)` | `concat("Vienna, ", "Austria")`|
-| Combines two or more strings. | "Vienna, Austria"              |
+## Syntax
 
-## Combining text from different columns
+```
+concat(value1, value2, ...)
+```
 
-| City     | Country | Location         |
-|----------|---------|------------------|
-| Vienna   | Austria | Vienna, Austria  |
-| Paris    | France  | Paris, France    |
-| Kalamata | Greece  | Kalamata, Greece |
+`value1`, `value2` ... can be columns or values. Metabase will convert non-string columns into strings before concatenating their values.
 
-where **Location** is a custom column with the expression:
+### Example
 
-```
-CONCAT([City], ", ", [Country])
-```
+| Expression                               | Result                   |
+| ---------------------------------------- | ------------------------ |
+| `concat("Vienna", "Austria")`            | `"ViennaAustria"`        |
+| `concat("Vienna", " is in " ,"Austria")` | `"Vienna is in Austria"` |
+| `concat([City], " is in " ,[Country])`   | `"Vienna is in Austria"` |
+
+### Metabase will use unformatted values for non-string columns
+
+When you use non-string columns in `concat`, Metabase will not apply any of the [formatting](../../../data-modeling/formatting.md) you applied to the columns when converting them to text.
+
+For example, if you formatted a floating point number to display only the first two decimal digits in the table results, the results of `concat` would still include additional decimal digits (if any) found in the raw results
```

Comment:
```suggestion
For example, if you formatted a number to display only the first two decimal digits in the table results, the results of `concat` would still include additional decimal digits (if any) found in the raw results.
```

Good example. Don't think we need to introduce the "floating point" concept here.